### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by Alfred-KInyua**
### Pr Objectives 

- Use preprocessed HTML file with embedded Ruby code.
- Use layouts and templates for shared content.

**Achievements** 

- Implemented the four required views, following the wireframes and extracting specific data for each view.
- Updated test cases to meet new project changes, following the TDD approach.
- Used partials to isolate reusable pieces of code.
- Used previously defined methods to meet project expectations.

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<h1>This is a post</h1>
```
Comment (SajjadAhmad14) [inline app/views/posts/index.html.erb]: Please uses `recent_comments` to render 5 most recent comments in the `app/views/posts/index.html.erb` page.

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1 +1,16 @@
-<h1> list of users </h1>
\ No newline at end of file
+<h1>list of users</h1>
+<div class="user">
+<%= render 'partials/user', user: @user %>
```
Comment (SajjadAhmad14) [inline app/views/users/show.html.erb:2]: Please follow [this](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_page.png?token=GHSAT0AAAAAAB2NSP7SPLVSZL5GNLPARTWUY2ZRPWQ) wireframe for the `user show` page.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Kinyua, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
Git flow is followed. :heavy_check_mark:
PR has a title and summary. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app/views/users/show.html.erb (hunk 2 above)
Comment (Alfred-KInyua) [reply app/views/users/show.html.erb:2]: @SajjadAhmad14 Thanks for that review, let me make the change

**Review by DammyShittu — APPROVED** [review]
Hi @Alfred-KInyua,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**